pkg/columnstore/part.go

```diff
@@ -9,11 +9,13 @@ type Row struct {
 	Values []interface{}
 }
 
+// Part is a sub set of a Granule? TODO: How do they relate?
```

Comment:
A part is an ordered set of rows that sort into this granule, but haven't yet been merged into the sorted portion of the Granule.